apps/web/pages/settings/security/two-factor-auth.tsx

```diff
@@ -34,14 +34,15 @@ const TwoFactorAuthView = () => {
   if (isLoading) return <SkeletonLoader />;
 
   const isCalProvider = user?.identityProvider === "CAL";
+  const canSetupTwoFactor = !isCalProvider && !user?.twoFactorEnabled;
```

Comment:
We need to check if 2FA is already enabled before disabling interacting with the configuration, otherwise converted CAL to GOOGLE users will not be able to disable 2FA